FirebaseInAppMessaging.podspec

```diff
@@ -38,6 +38,7 @@ See more product details at https://firebase.google.com/products/in-app-messagin
 	base_dir + "Sources/RenderingObjects/**/*.[cmh]",
 	base_dir + "Sources/Runtime/**/*.[cmh]",
 	base_dir + "Sources/Util/**/*.[cmh]",
+  base_dir + "Swift/Source/**/*.swift",
```

Comment:
nit: Consider making the indentation consistent here (lines 31 - 40 use tabs, line 41 uses 2 spaces, lines 42 - 44 use 4 spaces) while you're touching this file. Alternatively we could look into a ruby formatter in the future and handle them all at once.

Reply:
Yep, good call, thanks. Done!